Server/src/services/vehicle.service.ts

```diff
@@ -112,47 +107,59 @@ export default class VehicleService {
 			trackerLogs.push(trackerLog)
 		}
 
-		// initialize logs for apps and check if there are any
-		const appLogs = (await database.logs.getNewestLogs(vehicle.uid, 30)).filter(function (log) {
-			return log.trackerId == null
+		// initialize logs for apps and filter them, so we only have 10 at most from different points in time
```

Comment:
I really had trouble understanding the purpose of this code. Maybe you could write that the logs are at least 3s apart or just put a comment in front of the if-statement to explain this.